src/borg/helpers/msgpack.py

```diff
@@ -46,6 +46,7 @@
 """
 
 from .datastruct import StableDict
+from . import make_path_safe
```

Comment:
you do not use this in msgpack related code, you only imported it because you placed assert_normalized/to_normalized here (but you do not call them from here either), but they should be in `helpers.fs`also.

Reply:
Import removed